Phonebook.Frontend/src/app/pages/dashboard/components/team/team.component.ts

```diff
@@ -0,0 +1,68 @@
+import { Component, OnInit, OnDestroy } from '@angular/core';
+import { Select, Store } from '@ngxs/store';
+import { Observable, Subscription } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { Person, PhonebookSortDirection } from 'src/app/shared/models';
+import { BookmarksState } from 'src/app/shared/states';
+import { OrganigramService, UnitTreeNode } from 'src/app/services/api/organigram.service';
+import { CurrentUserService } from 'src/app/services/api/current-user.service';
+import { untilComponentDestroyed } from 'ng2-rx-componentdestroyed';
+import { Router } from '@angular/router';
+
+@Component({
+  selector: 'app-team',
+  templateUrl: './team.component.html',
+  styleUrls: ['./team.component.scss'],
+  host: { class: 'pb-expand' }
+})
+export class TeamComponent implements OnInit, OnDestroy {
+  public bookmarkedPersons: Person[];
```

Comment:
can we change the names of the variables to `team...`?

Reply:
These variables are actually for the bookmarks not the team. We need them so that when there are no bookmarks, it will say so even when the Team is shown.

Reply:
then please store only a bool. Because we don't need the person data. It's a smaller footprint. And maybe you rename this to `anyBookmarkedPeople`